lib/routes/account.js

```diff
@@ -972,6 +979,7 @@ module.exports = function (
       },
       handler: function (request, reply) {
         log.begin('Account.devices', request)
+        var language = request.query.language || 'en'
```

Comment:
Going back to this [fxa-js-client comment](https://github.com/mozilla/fxa-js-client/pull/212#issuecomment-239112038) - Having the auth server do the translation requires updates to 3 repos and seems very complex when we have a solution that could work content-server only.  Using jquery-timeago's built in singular/plural handling is substandard from an l10n perspective because it doesn't follow their best practices, but they have been a very understanding bunch when we explain the situation to them on the mailing list.